docs/INSTALL.md

```diff
@@ -89,14 +89,19 @@ Install Cmake (**3.22.1** minimum), Boost, Qt 6 and other dependencies.
 
 Installing *ccache* can speed up active development.
 
-_Ubuntu 20.04:_
+_Ubuntu 20.04 or older:_
 
-Because Ubuntu 20.04 only offers Qt 5, you need to add a PPA to get Qt 6. This will install Qt 6.2 but any other minor version should work as well.
+Because Ubuntu 20.04 and older versions only offer Qt 5, you need to add a PPA to get Qt 6. This will install Qt 6.2 but any other minor version should work as well.
 
 ```bash
 sudo add-apt-repository -y ppa:savoury1/qt-6-2
 ```
 
+Furthermore the minimum required `geoclue` version is `2.5.7` which is newer than,
```

Comment:
Let's disable the geoclue by default to avoid crashes for contributors.

Reply:
Well, this would be a documentation change here to inform the users. I believe this change is needed regardless of the request.
In the theoretical scenario in which what you requested is doable and also reasonable, the implementation would go into a separate PR.